asf_welsh_energy_consultation/getters/get_data.py

```diff
@@ -19,99 +19,24 @@
-
-
-def create_argparser():
-    """
-    Creates an Argument Parser that can receive the following arguments:
-    - local_data_dir
-    - epc_batch
-    - mcs_batch
-
-    Returns:
-        Argument Parser
-    """
-    parser = ArgumentParser()
-
-    parser.add_argument(
-        "--local_data_dir",
-        help="Local directory where EPC data is/will be stored",
-        type=str,
-    )
-
-    parser.add_argument(
-        "--supp_data",
-        help='Name of directory where supplementary data is stored in the form `data_YYYYMM`. Defaults to "newest"',
-        default="newest",
-        type=str,
-    )
-
-    parser.add_argument(
-        "--epc_batch",
-        help='Specifies which EPC data batch to use in the form `YYYY_[Quarter]_complete`. Defaults to "newest"',
-        default="newest",
-        type=str,
-    )
-
-    parser.add_argument(
-        "--mcs_batch",
-        help="Specifies which MCS installations data batch to use. Only date required in YYMMDD format. "
-        "Defaults to 'newest'",
-        default="newest",
-        type=str,
-    )
-
-    parser.add_argument(
-        "--calculate_average_installations",
-        help="Calculate additional high level statistics specific for October 2023 analysis. Defaults to 'False'",
-        default=False,
-        type=bool,
-    )
-
-    return parser
-
-
-def get_args():
-    """
-    Get arguments from Argument Parser.
-
-    Returns:
-        List of arguments.
-    """
-    parser = create_argparser()
-
-    args = parser.parse_args()
-
-    if args.supp_data == "newest":
-        subdirs = [
-            subdir for subdir in os.listdir(os.path.join(PROJECT_DIR, supp_data_dir))
-        ]
-        args.supp_data = max(subdirs)
-        logger.info(
-            f"Using supplementary folder from the following directory: {os.path.join(PROJECT_DIR, supp_data_dir, max(subdirs))}"
-        )
-
-    return args
-
-
-arguments = get_args()
-LOCAL_DATA_DIR = arguments.local_data_dir
-input_data_path = os.path.join(supp_data_dir, arguments.supp_data)
-wales_epc_path = "wales_epc.csv"
-
-
-def get_mcs_and_joined_data(epc_version):
+def get_mcs_and_joined_data(epc_version, mcs_date):
     """
     Get cleaned MCS data, and cleaned MCS data fully joined with EPC dataset, and cleaned MCS data joined with most recent EPC before HP
     installation or earliest after installation, up to date specified in args.
 
+    Args:
+        epc_version (str): specifies which EPC version is joined to MCS data. One of "none", "full", "newest" or "most_relevant".
+            "none" returns just installation data, "full" returns installation data with
+            each property's entire EPC history attached, "newest" selects the EPC
+            corresponding to the most recent inspection and "most_relevant" selects the
+            most recent EPC from before the HP installation if one exists or the earliest EPC
+            from after the HP installation otherwise. Defaults to "none".
```

Comment:
Just thought it might make everything a little clearer.

```suggestion
        epc_version (str): specifies which EPC version is joined to MCS data. One of "none", "full", "newest" or "most_relevant".
            "none" returns just MCS installation data, "full" returns MCS installation data with
            each property's entire EPC history attached, "newest" selects the EPC
            corresponding to the most recent inspection and "most_relevant" selects the
            most recent EPC from before the HP installation if one exists or the earliest EPC
            from after the HP installation otherwise. Defaults to "none".
```